backend/apps/readings/analysis.py

```diff
@@ -4,9 +4,96 @@
 
 """
 import statistics
+from pathlib import Path
+
+from config.settings.base import PROJECT_ROOT
 from .models import StudentResponse
 
 
+def compare_abs_value(val1, val2):
```

Comment:
maybe `max_abs` ? In which case, if you're returning the max, in the event of a tie just return the value. (plus if you implement this using `max` then you cut it down to 3 lines). Am I missing something where you need the 0?